CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Add the ``configureActionButtons`` method to the AdminInterface
 - Add the ``configureActionButtons`` method to the AdminExtensionInterface
+- Add the ``getAccessMapping`` method to the AdminExtensionInterface
```

Comment:
sorry this is my fault, i thought we use only one tick in this class, but we use two, so everything is fine!